lib/sycamore/sycamore/data/element.py

```diff
@@ -204,6 +200,16 @@ def tokens(self) -> Optional[list[dict[str, Any]]]:
     def tokens(self, tokens: list[dict[str, Any]]) -> None:
         self.data["tokens"] = tokens
 
+    @property
+    def text_representation(self) -> Optional[str]:
+        if not self.data["text_representation"] and self.data["table"]:
+            self.data["text_representation"] = self.data["table"].to_csv()
+        return self.data["text_representation"]
+
+    @text_representation.setter
+    def text_representation(self, text_representation: str) -> None:
+        self.data["text_representation"] = text_representation
```

Comment:
Can we just inherit this from Element?

Reply:
We'll need to redefine a setter since we're overriding the getter method.

Reply:
We'll need to override both to keep the property as read-write instead of read-only